finance/bitcoin/Portfile

```diff
@@ -1,11 +1,10 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
-PortGroup               cxx11 1.1
 
 name                    bitcoin
 categories              finance crypto
-version                 0.17.1
+version                 0.19.0.1
 revision                3
```

Comment:
the ```revision``` needs to be reset to ```0``` when updating the version

Reply:
Thanks, have fixed that up.